From d57fb7df07a8c793095e243fe69a14b83b68558d Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sun, 28 Aug 2022 16:00:26 +0200
Subject: [PATCH 175/391] arm64: dts: rk3399-pinebook-pro: Fix codec frequency
 after boot

Without assigned clocks, codec mclk would have too high frequency right
after boot.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
index 6b094a1ec..7f60bf059 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
@@ -688,6 +688,8 @@ es8316: es8316@11 {
 		reg = <0x11>;
 		clocks = <&cru SCLK_I2S_8CH_OUT>;
 		clock-names = "mclk";
+		assigned-clocks = <&cru SCLK_I2S_8CH_OUT>, <&cru SCLK_I2S1_8CH>;
+		assigned-clock-rates = <11289599>, <11289599>;
 		#sound-dai-cells = <0>;
 	};
 };
-- 
2.35.3

